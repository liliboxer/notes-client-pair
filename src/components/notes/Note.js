import React from 'react';
import PropTypes from 'prop-types';

function Note({ note }) {
  return (
    <section>
      <h3>{note.title}</h3>
      <p>{note.body}</p>
    </section>
  );
}

Note.propTypes = {
  note: PropTypes.object.isRequired
};

export default Note;
