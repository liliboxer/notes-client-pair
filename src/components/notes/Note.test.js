import React from 'react';
import { shallow } from 'enzyme';
import Note from './Note';

describe('Note component', () => {
  it('renders Note', () => {
    const wrapper = shallow(<Note note={{}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
  
