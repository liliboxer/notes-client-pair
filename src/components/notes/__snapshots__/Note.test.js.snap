// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders Note 1`] = `
<section>
  <h3 />
  <p />
</section>
`;
